### DISCUSSION TITLE
CDM Versions

### DISCUSSION THREAD

**Discussion by Teyiowuawi** [finos/common-domain-model#3010] [Q&A] [answered]
Hi, 

anyone else having some trouble viewing the CDM in other versions. 

I am trying to view the CDM as JSON schema (https://cdm.finos.org/docs/download/#cdm-as-json-schema). I have downloaded the zip file for the latest version but when i try to extract the files, I get a pop up saying the folder is empty. 

Can anyone help please?
![image](https://github.com/finos/common-domain-model/assets/121977243/a4de503e-62d5-4e8d-a655-0e1da85bc0a5)

**Comment by chrisisla** [comment, marked as answer]
Hi @Teyiowuawi,

I had something similar when trying to use Windows, but I now use [7zip ](https://www.7-zip.org/ ) which works fine. Once you open the zip you'll see a compressed file within it which you'll need to double click/open to see the goodies within it:

![image](https://github.com/finos/common-domain-model/assets/112546065/5f6ef378-45e1-4ed8-8b6f-9acdf19c033d)

![image](https://github.com/finos/common-domain-model/assets/112546065/278b9b39-d5c9-4234-af52-0634016de034)

![image](https://github.com/finos/common-domain-model/assets/112546065/29aa1dd0-389d-4013-b5a3-ccf986fc5ed8)

So I guess the answer is to try 7zip or another zip/archive manager of your choice, rather than use what Windows offers.

**Reply by Teyiowuawi** [reply]
Hi @chrisisla 

Tried that and it works a charm, thank you!
